packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,71 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface State {
+  inode?: Inode;
+  other_elts?: any;
+  inode_extender?: InodeExtender;
```

Comment:
I am not sure other_elts and inode_extender are quite right. inode_extender seems to be a property inside other_elts not at the same level.